src/category_theory/limits/limits.lean

```diff
@@ -1081,27 +1116,30 @@ has_colimits.has_colimits_of_shape J
 
 /- Interface to the `has_colimit` class. -/
 
-/-- The chosen colimit cocone of a functor. -/
-def colimit.cocone (F : J ⥤ C) [has_colimit F] : cocone F := has_colimit.cocone
+/-- An arbitrary choice of colimit cocone of a functor. -/
+def colimit.cocone (F : J ⥤ C) [has_colimit F] : cocone F := (get_colimit_cocone F).cocone
 
-/-- The chosen colimit object of a functor. -/
+/-- An arbitrary choice of colimit object of a functor. -/
 def colimit (F : J ⥤ C) [has_colimit F] := (colimit.cocone F).X
 
-/-- The coprojection from a value of the functor to the chosen colimit object. -/
+/-- The coprojection from a value of the functor to the colimit object. -/
```

Comment:
I feel like `chosen` here is still appropriate - as in the previous docstring, the colimit is an arbitrary choice so it is still a "chosen colimit object".

Reply:
(and idem below)

Reply:
My hesitancy here is that if we do introduce `chosen_limit` soon, this will then become misleading, and it seems fine without the word `chosen` here.